tests/cache/models/samples/post.js

```diff
@@ -257,7 +255,30 @@ describe('tests/cache/models/samples/post.js >', () => {
         }
 
         expect(res.body.name).to.be.equal(sampleName);
-        done();
+        return done();
+      });
+    });
+
+    it('basic post /samples with invalid token', (done) => {
+      api.post(path)
+      .set('Authorization', 'invalidtoken')
```

Comment:
are we supposed to expect the POST to succeed with invalid token? this test wasn't in the original  tests/cache/models/samples/post.js file

Reply:
This test was missed in the original file. It covers line 232 in api/v1/controllers/samples.js now and would have covered line 43 in the pervious version of api/v1/helpers/verbs/postUtils.js